src/main/kotlin/rx/lang/kotlin/observables.kt

```diff
@@ -1,12 +1,14 @@
 package rx.lang.kotlin
 
+import rx.AsyncEmitter
 import rx.Observable
 import rx.Subscriber
 import rx.Subscription
 import rx.observables.BlockingObservable
 
 fun <T> emptyObservable() : Observable<T> = Observable.empty()
 fun <T> observable(body : (s : Subscriber<in T>) -> Unit) : Observable<T> = Observable.create(body)
+fun <T> fromAsync(body: (s: AsyncEmitter<in T>) -> Unit, mode: AsyncEmitter.BackpressureMode): Observable<T> = Observable.fromAsync<T>(body, mode)
```

Comment:
What does this add over just an import of the static method directly from `Observable`?